mongo/hatch.toml

```diff
@@ -0,0 +1,26 @@
+[env.collectors.datadog-checks]
+
+[envs.default]
+dependencies = [
+  "datadog_checks_tests_helper @ {root:uri}/../datadog_checks_tests_helper",
+]
+
+[[envs.default.matrix]]
+python = ["38"]
+version = ["4", "5"]
```

Comment:
```suggestion
version = ["4.4", "5.0"]
```

Reply:
I can change that, no problem but is there a particular reason you prefer having `4.0` instead of `4`?

Reply:
sounds good, will also apply that to the other PRs as well